src/controllers/keywords.controller.js

```diff
@@ -5,6 +5,10 @@ exports.searchVideos = async function (req, res, next) {
   const { userInput } = req.body;
   const userQuery = userInput.join(" ");
 
+  if (!userQuery) {
+    return res.status(200).send({ result: "ok", videos: [], query: userQuery });
+  }
+
```

Comment:
```suggestion
  if (!userQuery) {
    res.status(200).send({ result: "ok", videos: [], query: userQuery });
    return;
  }
```
위에처럼 작성하면 consistent-return 규칙을 따를 수 있을 거 같아요!

Reply:
말씀처럼 변경하니 규칙추가가 필요 없네요! 수정하겠습니다!